packages/legend-query-builder/src/stores/fetch-structure/tds/QueryBuilderTDSState.ts

```diff
@@ -121,8 +121,9 @@ export class QueryBuilderTDSState
   isConvertDerivationProjectionObjects = false;
   showPostFilterPanel: boolean;
   showWindowFuncPanel = false;
-  draggedColumnIndex: number | undefined;
   hoveredColumnIndex: number | undefined;
```

Comment:
we don't need this anymore, let's remove it, we can easily figure out the swap index based on the gap :)